bsp/esp32_s3_usb_otg/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.0.0~1"
+version: "2.0.1"
```

Comment:
`ADC_ATTEN_DB_12` was introduced in v5.1.3. We must bump the required version of idf in this component

Reply:
https://github.com/espressif/esp-bsp/blob/fd3b8ba38ab204d8607ff4de2a3ecf37ca53d83c/bsp/esp32_s3_usb_otg/esp32_s3_usb_otg_idf4.c can be removed and CMakelists.txt updated